certificate-policy.md

```diff
@@ -150,7 +96,7 @@ As defined in Section 1.6.1.
 
 
 ### 1.3.4 Relying Parties
-"Relying Party" and "Application Software Supplier" are defined in Section 1.6.1. Current Members of the CA/Browser Forum who are Application Software Suppliers are listed here: https://cabforum.org/members.
+"Relying Party" and "Application Software Supplier" are defined in Section 1.6.1. 
 
 
 ### 1.3.5 Other Participants
```

Comment:
Currently, the other participants text is specific to the BRD development.  I think we should update AFTER we iterate, to ensure inclusiveness.  

> ### 1.3.5 Other Participants
> Other groups that have participated in the development of these Requirements include the AICPA/CICA WebTrust for Certification Authorities task force and ETSI ESI. Participation by such groups does not imply their endorsement, recommendation, or approval of the final product.